src/executables/master.c

```diff
@@ -375,6 +380,12 @@ css_accept_new_request (CSS_CONN_ENTRY * conn, unsigned short rid, char *server_
 		    }
 		}
 	    }
+	  if (!entry->ha_mode)
```

Comment:
ha_mode를 배제하는 이유가 뭐였었죠?
```suggestion

	  if (!entry->ha_mode)
```

Reply:
ha server의 경우에는 heartbeat 쪽에서 server revive를 위해 별도의 entry list를 관리하고 있기 때문에, `master_Server_monitor`에서는 non-HA server의 entry만을 저장하여야 합니다.